### DISCUSSION TITLE
Some cameras not sending motion notifications

### DISCUSSION THREAD

**Discussion by saltyzoo** [dgreif/ring#1236] [General]
This is more of an FYI for others. I've been beating my head against the wall trying to figure out why 3 out of my 12 cameras refused to send notifications to the api. I compared every setting to working cameras and could not find any pattern, nothing worked.

Finally, in desperation I went to the iPhone app and disabled motion detection and re-enabled it, and viola, notifications. Not sure why, but it worked for me.

**Comment by dgreif** [comment]
This sounds like it might be caused by #1218. I've released 11.8.0 which should fix this issue.

**Comment by saltyzoo** [comment]
I was running 11.8.0 beta at the time.

**Reply by dgreif** [reply]
Very interesting 😕. It sounds like this is working for you now, so nothing to do immediately, but I'll keep an eye out for similar reports

**Comment by raiubreaksthings** [comment]
Think I am seeing a similar issue. Starting to think it might be a TvOS issue. I can see in the logs that motion event was detected, pulling snapshot before sending notification. The HomeKit notification either does not show on the Apple TV or iPhone, or does show but stale. The stale snapshot then seems to lock the Apple TV up for a bit. I am seeing this happen when using Scrypted in docker as well. 

I have done several fresh installs of ubuntu, homebridge, and ring plug in.
 The notifications work for a bit and then start to experience the issue. 

Apple TV 4K 1st gen TvOS 17.1
Dell OptiPlex i7 running ubuntu basic 24 only running homebridge for now.
Homebride v1.7
 Node.js Version	v20.10.0

**Comment by lucidmattnyc** [comment]
I am having the same issue.  I've completed all the steps seen elsewhere, removing the bridge from Homekit, removing from HomeBridge, removing all authorized devices (including primary iOS) device, reinstalling to Homebridge and setup as child bridge, adding cams to HomeKit and no notifications on any device (iOS, MacOS, or AppleTV).  Homebridge can see the notification, but nothing seems to happen regarding pushing it to HomeKit.
